### ISSUE TITLE
Import snapshot directly into hotstore for nodes running splitstore

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/lotus#10554]
### Checklist

- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

Coldstore shouldn't track unwanted state in discard mode.  Today we waste ~100 G of disk space no matter what when syncing from snapshot 

### Describe the solution you'd like

import-snaphsot daemon flag should import the chain into the hotstore if the node is configured to run with splitstore.

Basically repo should create a blockstore using hotstore instead of universal store location if applicable.  We might want to make a new blockstore location that automatically does this or we could manually check the repo config during import.

### Describe alternatives you've considered

We could also make it easier to purge the coldstore of snapshot state by improving chain prune cold-compact but it would be better to prevent this problem in the first place than to keep it around and also create a solution

**Comment by ZenGround0** [comment]
Looking at this more today the problem seems to be that 
1. Node startup depends on initial state in universal blockstore 
2. Splitstore startup depends on node startup

So we have to unravel these dependencies if we want to save the disk space
